sysutils/port-whatsnew/Portfile

```
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        macports macports-contrib 29259cdad5f470bb1ebbcf3627f498aa7b2dfb2b
+github.tarball_from archive
+
+name                port-whatsnew
+version             0.1
+revision            0
+categories          sysutils macports
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           any
+license             BSD
+supported_archs     noarch
+
+description         Show the log from the MacPorts version control system of \
+                    what changed since this port was installed. This script was \
+                    originally written when the MacPorts VCS was still svn, but \
+                    the intention is to migrate it to use git, just like the \
+                    MacPorts VCS has done.
+long_description    ${description}
+homepage            https://github.com/${github.author}/${github.project}/blob/master/${name}
+
+livecheck.type      none
+
+fetch.type          git
+
+worksrcdir          ${name}
+
+use_configure       no
+
+build {
+    ui_debug "${name} has no build step."
+}
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/${name}/${name} ${destroot}${prefix}/bin/${name}
```

0755 is the default mode and the default destination name is the same as the source name.
```suggestion
    xinstall ${worksrcpath}/${name}/${name} ${destroot}${prefix}/bin/
```